lib/api/createFile.js

```diff
@@ -114,7 +114,7 @@ module.exports = function createFile(bucketId, fileName, file, opts, cb) {
 // isn't one already
 function fileToStream(file) {
   // Is it already a readable stream?
-  if(file instanceof stream.Readable) {
+  if(file.readable) {
```

Comment:
Agreed, though we do this check in other places, particularly core, so we may want to move over there in the long run. Sent from Android